src/layout/layout.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Drawer,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  Toolbar,
+  ListItemText,
+} from "@mui/material";
+import {
+  Settings as ConfigIcon,
+  Description as SchemaIcon,
+} from "@mui/icons-material";
+import { useNavigate } from "react-router-dom";
+import { routes } from "../routing/routes";
+
+type LayoutProps = {
+  children: React.ReactNode;
+};
+
+type DrawerItem = {
+  text: string;
+  icon: React.ReactNode;
+  onClick: () => void;
+};
+
+export const Layout: React.FC<LayoutProps> = ({ children }) => {
+  const navigate = useNavigate();
+
+  const drawerItems: DrawerItem[] = [
+    {
+      text: "Config",
+      icon: <ConfigIcon />,
+      onClick: () => navigate(routes.CONFIG),
+    },
+    {
+      text: "Schema",
+      icon: <SchemaIcon />,
+      onClick: () => navigate(routes.SCHEMA),
+    },
+  ];
+  return (
+    <div>
```

Comment:
In the persistent drawer example they used the `Box` component, we can use it as well.

Reply:
excellent idea!